doc/mobile_base_arm/mobile_base_arm_tutorial.rst

```diff
@@ -0,0 +1,108 @@
+Planning For Differential-Drive Mobile Base and an Arm
+======================================================
+.. image:: images/intro.png
+   :width: 700px
+
```

Comment:
I'd add a bit of an intro before getting into "Getting Started". Perhaps something like this:
```
This tutorial will show how you can configure a mobile manipulator with a differential drive base so that you can plan coordinated motions for the base and the other degrees of freedom using MoveIt. While you could use MoveIt to plan for just the base, this is not recommended. Use Nav2 for pure navigation. Note that the tutorial uses Hello Robot's Stretch platform for demonstration purposes, the same approach can be used for any mobile manipulator with a diff-drive base. 
```

In the into image, it looks like the gripper is part of the base. Can you put the arm and gripper up a bit, so that it's obvious that it's attached to the torso and not the base?